src/DependencyInjection/Configuration.php

```diff
@@ -251,6 +251,50 @@ private function addFilesystemSection(ArrayNodeDefinition $node): void
                             ->end()
                         ->end()
 
+                        ->arrayNode('spaces')
```

Comment:
Can you please try to extract some small methods. This will improve readability.